system/forms/formbuilder/item-types/url.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form i18nBaseUri="formbuilder.item-types.url:">
+	<tab id="default">
+		<fieldset id="default">
+			<field name="protocol" control="select" values="http://,https://" labels="http://,https://" defaultvalue="" />
```

Comment:
This feels a little tough to explain for admin users. I'd rather we didn't make this a configurable option at this point to keep the system simpler. i.e. limiting user input to a specific protocol isn't all that helpful(?)